spec/models/game_spec.rb

```diff
@@ -162,7 +162,8 @@
     end
 
     it 'returns all list items belonging to the game' do
-      items = game.shopping_lists.map { |list| list.list_items.to_a }.flatten.sort
+      items = game.shopping_lists.map {|list| list.list_items.to_a }
+                .flatten.sort
```

Comment:
This is actually pretty ugly the way it doesn't align with anything, but I guess that's how that cop works :/